### PR TITLE
Handle trashing files

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) Defect 732: As Messaging Service, I do not include trashed files when reading folder file and I respond with an error when reading a trashed file.](https://trello.com/c/2BOmVSyz/4434-2-defect-732-as-messaging-service-i-do-not-include-trashed-files-when-reading-folder-file-and-i-respond-with-an-error-when-readi)

File: README.md [hunk 1]
```diff
@@ -7,3 +7,22 @@ Rise Vision Messaging Service
...
+```bash
+npm install
+```
+
+* Set up `GOOGLE_APPLICATION_CREDENTIALS`
+
+* Test:
+```bash
+npm run test
+```
```
Comment (santiagonoguez) [inline README.md]: 👍

File: src/gcs.js [hunk 2]
```diff
@@ -38,21 +44,25 @@ module.exports = {
...
+      fields: "items(name,generation,metadata)"
     })
     .then(files=>{
       if (files[0].length === 0) {
         return Promise.reject(Error("NOEXIST"))
       }
 
-      const nonFolderFiles = files[0]
-      .filter(file=>!file.name.endsWith("/"));
+      const folderFiles = files[0]
```
Comment (santiagonoguez) [inline src/gcs.js]: @andrecardoso why the variable nonFolderFiles was renamed to folderFiles, if we are still selecting files that are not folders ( that is, files that don't end with '/' ) ?

Comment (andrecardoso) [reply src/gcs.js]: @santiagonoguez we're now selecting non folder and non trashed entries, so I thought `nonFolderFiles` could be misleading, but maybe something like `nonFolderNorTrashedFiles` is better

Comment (santiagonoguez) [reply src/gcs.js]: @andrecardoso or maybe something like `availableRegularFiles` or `regularFiles` if you really want to avoid the 'non' prefix.

**Pull request merged**